apps/fm/js/fm.js

```diff
@@ -184,11 +184,24 @@ function updatePowerUI() {
 }
 
 function updateAntennaUI() {
-  $('antenna-warning').hidden = mozFMRadio.antennaAvailable;
```

Comment:
But there are cases where both warnings are on and visible at the same time, so shouldn't we only be showing one of those warnings?